frame/contracts/src/storage.rs

```diff
@@ -103,8 +130,8 @@ where
 	}
 
 	/// Returns `true` iff the `key` exists in storage.
```

Comment:
```suggestion
	/// Returns `Some(len)` (in bytes) if a storage item exists at `key`.
	///
	/// Returns `None` if the `key` wasn't previously set by `set_storage` or
	/// was deleted.
```

Copy-pasting from the `get_storage_size` docs